### PR TITLE
Bump deps, and clean up some imports

### PR DESCRIPTION
**Pull request by grebe**
Java 13 isn't cooperating. @ucbjrl, does this look OK?

### CODE REVIEW

File: rocket/src/main/scala/craft/ShiftRegisterMem.scala [hunk 1]
```diff
@@ -3,7 +3,7 @@
 package craft
 
 import chisel3._
-import chisel3.core.requireIsHardware
+import chisel3.internal.requireIsHardware
```
Comment (ucbjrl) [inline rocket/src/main/scala/craft/ShiftRegisterMem.scala]: Shouldn't this be `import chisel3.experimental.requireIsHardware`?

File: rocket/src/main/scala/dspblocks/MemToRegmap.scala [hunk 2]
```diff
@@ -1,7 +1,7 @@
 package dspblocks
 
 import chisel3._
-import chisel3.core.requireIsChiselType
+import chisel3.internal.requireIsChiselType
```
Comment (ucbjrl) [inline rocket/src/main/scala/dspblocks/MemToRegmap.scala]: Shouldn't this be `import chisel3.experimental.requireIsChiselType`?

File: rocket/src/main/scala/craft/ShiftRegisterMem.scala (hunk 1 above)
Comment (grebe) [reply rocket/src/main/scala/craft/ShiftRegisterMem.scala]: Should it? This is what the deprecation warning chisel3.core gave me. I'll switch if experimental is preferred

Comment (ucbjrl) [reply rocket/src/main/scala/craft/ShiftRegisterMem.scala]: Hmm. Maybe I'm out of step. The `experimental` piece may be a compatibility bone. If the warning says to use `internal`, then I'd use it.

**Pull request merged**